design/work_item_link.go

```diff
@@ -125,7 +125,6 @@ var workItemLinkList = JSONList(
 var _ = a.Resource("work_item_link", func() {
 	a.BasePath("/workitemlinks")
 	a.Action("show", showWorkItemLink)
```

Comment:
@xcoulon Shouldn't we remove any other reference to `listWorkItemLinks`  in the design file ?